charts/rasa-x/templates/nginx-deployment.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     {{- include "rasa-x.labels" . | nindent 4 }}
     app.kubernetes.io/component: nginx
 spec:
-  replicas: {{ default 1 .Values.nginx.replicaCount }}
+  replicas: 1
```

Comment:
may I ask why this is now hardcoded to 1?

Reply:
Sure! Both components (rasa-x, nginx) don't make a lot of sense to scale at the moment. I can also re-add it, but I would still not recommend doing. Scaling nginx should not result in any problems though. Will re-add it 👍

Reply:
Yeah scaling nginx should not be a problem. For rasa-x you are probably right about rasa-x though